src/index.js

```diff
@@ -0,0 +1,171 @@
+import './styles.css';
+import { ShowElement, HideElement } from './visiable.js';
+import CheckIfCompleted from './completion.js';
+
+let tasks = [];
+
+const enterIcon = document.getElementById('enter-icon');
+const addField = document.getElementById('add-field');
+const cleanBtn = document.getElementById('clean-btn');
+const listContainer = document.getElementById('list-container');
+let ShowList;
+
+if (localStorage.getItem('to_do_list') !== null) {
+  tasks = JSON.parse(localStorage.getItem('to_do_list'));
+}
+
+const UpdateStorage = () => localStorage.setItem('to_do_list', JSON.stringify(tasks));
+
+const UpdateIndex = (x) => {
+  for (let i = x; i < tasks.length; i += 1) {
+    tasks[i].index = i;
+  }
+  UpdateStorage();
+  ShowList();
+};
+
+const EditFun = (varX) => {
+  const editArrItem = tasks.filter((item) => item.index === varX);
+  return editArrItem[0].description;
+};
+
+const DelFun = (varX) => {
+  tasks = tasks.filter((item) => item.index !== varX);
+};
+
+const OkFun = (varX, val) => {
+  tasks = tasks.filter((item) => {
+    if (item.index === varX) {
+      item.description = val;
+    }
+    return true;
+  });
+};
+
+const CheckFun = (varI, check) => {
+  tasks[varI].completed = CheckIfCompleted(check);
+};
+
+ShowList = () => {
+  listContainer.innerHTML = '';
+  for (let i = 0; i < tasks.length; i += 1) {
+    const listItem = document.createElement('li');
+    const checkIcon = document.createElement('input');
+    const span = document.createElement('span');
+    const editField = document.createElement('input');
+
+    editField.style.height = '3rem';
+    editField.style.outline = 'none';
+    editField.style.borderColor = 'gainsboro';
+    editField.style.padding = '5px';
```

Comment:
Kindly consider adding all your styling into the `style.css` file as a best practice instead of JavaScript files.